src/CONFIG.js

```diff
@@ -1,51 +1,53 @@
 import _ from 'underscore';
+import lodashGet from 'lodash.get';
 import {Platform} from 'react-native';
 import Config from 'react-native-config';
+import getPlatform from './libs/getPlatform/index';
+
+// Set default values to contributor friendly values to make develo:pment work out of the box without an .env file
+let expensifyCashURL = lodashGet(Config, 'URL_EXPENSIFY_CASH', 'https://expensify.cash/');
+let expensifyURL = lodashGet(Config, 'EXPENSIFY_URL_COM', 'https://www.expensify.com/');
+let ngrokURL = lodashGet(Config, 'NGROK_URL', null);
```

Comment:
NAB, can probably leave off the `null` since we will get `undefined` without a default arg in `get()` and that'd work too.